lib/expression/eval.ex

```diff
@@ -202,6 +202,10 @@ defmodule Expression.Eval do
 
   # Support comparing a `Date`/`DateTime` value with an ISO8601 date/datetime string
   # credo:disable-for-next-line Credo.Check.Refactor.CyclomaticComplexity
+  def op(operator, a, b) when operator == :== and is_nil(a) and is_nil(b) do
```

Comment:
Let's include `:=` for backward compatibility.
```suggestion
  def op(operator, a, b) when operator in [:=, :==] and is_nil(a) and is_nil(b) do
```